science/projects/soroush_heidary/01-main-content/Main-Content.ipynb

```diff
@@ -0,0 +1,419 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        content = base64.b64decode(html)
</pre><p>I recall that you had previously asked about my use of Latin-1 encoding in the Try-Except statement for decoding HTMLs. The reason for this is because certain Unicode characters were unable to be decoded, causing errors, as you have also experienced. While you handle these errors through replacement, I have taken a different approach by ignoring them altogether by flat. </p><p> </p><p>html_string&nbsp;=&nbsp;base64.b64decode(html.encode("utf-8",&nbsp;'ignore'))&nbsp;#&nbsp;type:&nbsp;byte</p><p>html_doc&nbsp;=&nbsp;html_string.decode("utf8",&nbsp;'ignore')&nbsp;#&nbsp;type:&nbsp;string.&nbsp;To&nbsp;avoid&nbsp;unicode&nbsp;errors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Zarebin/nextgen-06/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>